app/serializers/resource_tool_group_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ResourceToolGroupSerializer < ActiveModel::Serializer
+  attributes :resource_id, :tool_group
```

Comment:
don't include the `resource_id` or `tool_group` here

Reply:
excluded the attributes :resource_id & :tool_group